src/app/components/Footer/index.tsx

```diff
@@ -67,12 +68,12 @@ const renderColumn3 = () => <Styled.Column3>
 </Styled.Column3>;
 
 const renderSocialDirectory = () => <Styled.Social role='directory'>
-  <SocialIconMessage id='i18n:footer:social:fb:alt' href={fbUrl} Icon={Styled.FBIcon} />
-  <SocialIconMessage id='i18n:footer:social:tw:alt' href={twitterUrl} Icon={Styled.TwitterIcon} />
-  <SocialIconMessage id='i18n:footer:social:in:alt' href={linkedInUrl} Icon={Styled.LinkedInIcon} />
-  <SocialIconMessage id='i18n:footer:social:ig:alt' href={instagramUrl} Icon={Styled.IGIcon} />
+  <SocialIconMessage id='i18n:footer:social:fb:alt' href={fbUrl} target='_blank' Icon={Styled.FBIcon} />
+  <SocialIconMessage id='i18n:footer:social:tw:alt' href={twitterUrl} target='_blank' Icon={Styled.TwitterIcon} />
+  <SocialIconMessage id='i18n:footer:social:in:alt' href={linkedInUrl} target='_blank' Icon={Styled.LinkedInIcon} />
+  <SocialIconMessage id='i18n:footer:social:ig:alt' href={instagramUrl} target='_blank' Icon={Styled.IGIcon} />
```

Comment:
these social icons are all `target='_blank'`, you can just hardcode it inside the `SocialIconMessage` component